<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="SPgroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Sample Parameters</string>
    </property>
    <widget class="QLineEdit" name="input_length">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_thick">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_length">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Length (mm)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_thick">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Thickness (mm)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_def">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Displacement (mm)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_def">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_nCycles">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Cycles</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_nCycles">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="PTTgroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Resistances (Î©)</string>
    </property>
    <widget class="QLineEdit" name="input_ptt1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_ptt2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_ptt3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_ptt4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_ptt1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ptt1_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ptt1_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ptt1_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="PlotWidget" name="graph_pos" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>461</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="graph_resist" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>461</width>
      <height>211</height>
     </rect>
    </property>
    <widget class="QComboBox" name="select_resist">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Res0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Res1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Res2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Res3</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLineEdit" name="input_sLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_sLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sample Label</string>
    </property>
   </widget>
   <zorder>SPgroupBox</zorder>
   <zorder>PTTgroupBox</zorder>
   <zorder>btn_start</zorder>
   <zorder>btn_clear</zorder>
   <zorder>graph_pos</zorder>
   <zorder>graph_resist</zorder>
   <zorder>label_sLabel</zorder>
   <zorder>input_sLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>input_sLabel</tabstop>
  <tabstop>input_length</tabstop>
  <tabstop>input_thick</tabstop>
  <tabstop>input_def</tabstop>
  <tabstop>input_nCycles</tabstop>
  <tabstop>input_ptt1</tabstop>
  <tabstop>input_ptt2</tabstop>
  <tabstop>input_ptt3</tabstop>
  <tabstop>input_ptt4</tabstop>
  <tabstop>btn_start</tabstop>
  <tabstop>btn_clear</tabstop>
  <tabstop>select_resist</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
